app/src/main/java/com/shagalalab/constitution/article/ArticleFragment.kt

```diff
@@ -0,0 +1,47 @@
+package com.shagalalab.constitution.article
+
+import android.graphics.Typeface
+import android.os.Bundle
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.method.ScrollingMovementMethod
+import android.text.style.StyleSpan
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import com.shagalalab.constitution.data.ConstitutionDatabase
+import kotlinx.android.synthetic.main.fragment_article.*
+
+
+class ArticleFragment(private var chapterId: Int) :
+    Fragment(com.shagalalab.constitution.R.layout.fragment_article) {
+
+    companion object {
+        const val TAG = "ArticleFragment"
+    }
+
+    lateinit var viewModel: ArticleViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        viewModel =
+            ArticleViewModel(ConstitutionDatabase.getInstance(requireContext()).articleDao())
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        viewModel.getArticles(chapterId)
+        viewModel.articleList.observe(this, Observer { list ->
+            val s = SpannableStringBuilder()
+            list.forEach {
+                val st = StyleSpan(Typeface.BOLD)
```

Comment:
Is there any value of having this inside the loop? Would it be more efficient if we move it outside of the loop?